MediaGalleryUi/view/adminhtml/web/js/image/image-actions.js

```diff
@@ -27,6 +37,109 @@ define([
             }
 
             modalElement.modal('closeModal');
+        },
+
+        /**
+         * Delete image action
+         *
+         * @param {Object} record
+         */
+        deleteImageAction: function (record) {
```

Comment:
Can the deleteImageAction be extracted to a separate file and reused here and in `MediaGalleryUi/view/adminhtml/web/js/grid/columns/image/actions.js` to avoid code duplication?

Reply:
I think it is possible. I used action component reloadGrid method to refresh content at the media gallery grid and it works. I will try this today and report you back.
I didn’t use deleteImageAction because though that even if they do the same it is better to keep it separate because of implementation specifics. At some lines the code is different. For example, message scope.